bentoml/cli/deployment.py

```diff
@@ -233,6 +233,30 @@ def deployment():
     @click.option(
         '--replicas',
         help='Number of replicas. Option applicable to platform: Kubernetes',
+        type=click.INT,
+    )
+    @click.option(
+        '--memory-size',
+        help='Memory size for lambda function. '
+        'Option applicable to platform: aws-lambda',
+        type=click.INT,
+        default=1024,
+    )
+    @click.option(
+        '--timeout',
+        help='function timeout for lambda function. '
+        'Option applicable to platform: aws-lambda',
+        type=click.INT,
+        default=6,
+    )
+    @click.option(
+        '--s3-path',
+        help='S3 path for storing AWS lambda deployment bundle'
```

Comment:
help=S3 path for storing BentoService artifacts when deploying to AWS lambda. This allows deploying large models that exceeded the limit of XXX MB default Lambda function package size. The limit for mode on S3 is 500MB.
Optional, applicable to platform: aws-lambda.